src/tests/ftest/util/MpioUtils.py

```diff
@@ -40,36 +42,42 @@ def mpich_installed(self, hostlist):
 
         try:
             # checking mpich install
-            checkmpich = subprocess.check_output(["ssh", hostlist[0], "which mpichversion"])
-
-            # Obtaning the location where mpich is installed by removing "mpichversion"
-            self.mpichinstall = checkmpich[:-13]
+            checkmpich = subprocess.check_output(["ssh", hostlist[0],
+                                                  "which mpichversion"])
 
+            # Obtaning the location where mpich is installed by
+            # removing "mpichversion"
```

Comment:
removing?

Reply:
this is simply explaining what "self.mpichinstall = checkmpich[:-17]" is doing. checkmpich's should be something like /path/bin/mpichversion, so this is simply removing that "bin/mpichversion". So rather I should change it to  - removing bin/mpichversion

Reply:
I think you meant running/executing mpichversion

Reply:
Will change.

Reply:
Yeah, explain the magic number 13 by providing an example of what it looks like and what you are chopping off